kitsune/sumo/api_utils.py

```diff
@@ -298,51 +253,35 @@ def has_object_permission(self, request, view, obj):
         return user == creator
 
 
-def PermissionMod(cls, permissions):
+PermissionListSerializer = None
+
+
+def PermissionMod(field, permissions=[]):
```

Comment:
Generally it's bad to use mutable values for default args. Are we sure this never gets modified in any way by this code or downstream code?